src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -210,45 +185,21 @@ function transformToTableData(dataPlaneOverviews: DataPlaneOverview[]): DataPlan
       certificate = t('data-planes.components.data-plane-list.certificate.none')
     }
 
-    // assemble the table data
-    const item: DataPlaneOverviewTableRow = {
+    const warnings = getWarnings(dataPlaneOverview, props.canUseZones)
```

Comment:
Not sure I’m following. I’m not sure this data can exist on Zones (our types are incomplete there, actually another `any`) and if it can, this kind of warning would be appropriate to surface in either context (i.e. Dataplanes **and** Zones). Why should we move this to Zones and remove it here?

Though I’m thinking about whether we need the `canUseZones` check. Can it be that a Dataplane has the data for this warning (i.e. a discovery subscription with `version.kumaDp.kumaCpCompatible: false` and a kuma.io/zone tag on an inbound) while not in global mode in the first place, @lahabana?

Reply:
Ok so from my perspective if thats not making sense at all, then we should definitely get this in now and come back later. I think the functionality mirrors what we had previously. 1 sec